customization-tests/swagger/README.md

```diff
@@ -33,6 +33,40 @@ license-header: MICROSOFT_MIT_SMALL
 add-context-parameter: true
 models-subpackage: implementation.models
 context-client-method-parameter: true
-customization-jar-path: target/bodycomplex-customization-1.0.0-beta.1.jar
-customization-class: fixtures.bodycomplex.customization.BodyComplexCustomization
+customization-class: BodyComplexCustomization
 ```
+
+### Customization
+```java
+public class BodyComplexCustomization extends Customization {
+    @Override
+    public void customize(LibraryCustomization customization) {
+        PackageCustomization implementationModels = customization.getPackage("fixtures.bodycomplex.implementation.models");
```

Comment:
How does the user specify import statements if this code requires some additional classes that have to be imported?

Reply:
The users don't have to specify the imports if they are not ambiguous. Otherwise they could write `import aa.bbb.ccc;" above the class definition. The code here is simply injected into a maven project that has `customization-base` as a dependency.